src/WebJobs.Script/Description/FunctionLogInfo.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    // Each function can get its own log stream.
+    // Static per-function logging information.
+    public class FunctionLogInfo
```

Comment:
Rename to FunctionLogger? Similar to the FunctionInstanceLogger rename I suggested above, but this one is not instance specific, so I think just calling it FunctionLogger makes sense.